src/datasets/commands/test.py

```diff
@@ -54,7 +55,9 @@ def register_subcommand(parser: ArgumentParser):
             "--save_info", action="store_true", help="Save the dataset infos in the dataset card (README.md)"
         )
         test_parser.add_argument(
-            "--ignore_verifications", action="store_true", help="Run the test without checksums and splits checks"
+            "--ignore_verifications",
+            action="store_true",
+            help="Run the test without checksums and splits checks. Deprecated: use --verification_mode instead.",
```

Comment:
```suggestion
            help="Run the test without checksums and splits checks.",
```
I think it's better to drop this sentence as there is no `--verification_mode` argument.